libs/core/src/main/java/org/elasticsearch/core/internal/io/IOUtils.java

```diff
@@ -278,4 +278,16 @@ public static void fsync(final Path fileToSync, final boolean isDir) throws IOEx
         }
     }
 
+    /**
+     * An IO operation with a single input.
+     * @see java.util.function.Consumer
+     */
+    @FunctionalInterface
+    public interface IOConsumer<T> {
```

Comment:
drive by comment, so ignore as you want: This does not look used and a `org.elasticsearch.common.CheckedConsumer` could do the same, just less explicit?

Reply:
I really just resynced the class with lucene. I can do it separately